flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -317,21 +314,22 @@ private static void checkAndCreateDirectory(File directory) throws IOException {
 			try {
 				FileUtils.deleteDirectory(instanceBasePath);
 			} catch (Exception ex) {
-				LOG.warn("Failed to instance base path for RocksDB: " + instanceBasePath, ex);
+				logger.warn("Failed to instance base path for RocksDB: " + instanceBasePath, ex);
```

Comment:
```suggestion
				logger.warn("Failed to delete base path for RocksDB: " + instanceBasePath, ex);
```

Not sure, maybe I don't understand what the code is doing here.